.github/workflows/pr-test.yml

```diff
@@ -66,7 +66,7 @@ jobs:
      
       - name: Publish Unit Test Results
         uses: docker://ghcr.io/enricomi/publish-unit-test-result-action:v1.6
-        if: always()
+        if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
we don't need this now - github.event_name == 'push'

Reply:
Got it, we have 
```
	on:
  push:	  push:
    branches:	    branches:
      - main	      - main
  pull_request:	  pull_request:
```

Reply:
it it to run this step on push event along with pull_request event.